components/WeeklyPickButton/WeeklyPickButton.tsx

```diff
@@ -18,34 +19,41 @@ type WeeklyPickButtonProps = {
  * @param props.team - The team name
  * @param props.src - The image source
  * @param props.isDisabled - Whether the button is disabled
+ * @param props.homeAway - Shows whether the team is home or away.
  * @returns The rendered weekly pick button.
  */
 const WeeklyPickButton: React.FC<WeeklyPickButtonProps> = ({
+  homeAway,
   team,
   src,
   isDisabled = false,
 }): JSX.Element => {
   return (
-    <div className="flex items-center">
-      <RadioGroupItem
-        value={team}
-        id={team}
-        disabled={isDisabled}
-        data-testid="team-radio"
-      />
-      <Label htmlFor={team} data-testid="team-label" disabled={isDisabled}>
-        <Image
-          src={src}
-          alt={team}
-          width={48}
-          height={48}
-          priority
-          data-testid="team-image"
-          className="h-12 w-12"
+    <>
+      <label htmlFor="home" className="capitalize" data-testid="home-away">
+        {homeAway}
+      </label>
+      <div style={{ direction: 'ltr' }} className="flex items-center mt-4">
+        <RadioGroupItem
+          value={team}
+          id={team}
```

Comment:
i didn't write this. it was copy and pasted to add my changes

Reply:
What do you mean by copy and pasted, like merge conflicts?

Reply:
i actually copied and pasted it. when i add in wrappers... in this case the <></> wrapper, i copy everything, delete, add the wrapper, then paste everything back inside. I see now I need to be careful with it to avoid confusion. The reason why I added the <></> is because i had to add {homeAway} on top and everything has to be inside the same parent element so i added the empty fragments